packages/react-native/src/Authenticator/Defaults/ResetPassword/styles.ts

```diff
@@ -0,0 +1,30 @@
+import { StyleSheet } from 'react-native';
+
+import { ResetPasswordStyle } from './types';
+
+const TEAL = '#317d95';
+
+export const styles: ResetPasswordStyle = StyleSheet.create({
+  container: {
+    width: '90%',
```

Comment:
Why is this set to `'90%'`

Reply:
Ah good catch. I added this to make the demo look better, but the parent container should be responsible for the layout. I'll remove it.